### ISSUE TITLE
Migrate nodes from the old cluster to the new cluster

### ISSUE THREAD

**Issue by nariai** [frazer-lab/cluster#33]
Paul,

As you can see, our current bottleneck is the number of nodes in the new cluster system, and hence we want to move a part of nodes in the old cluster to the new cluster as soon as possible (maybe eight nodes out of 16 nodes, as the first step).

Can you tell us what need to be done before migrating nodes?

Naoki

**Comment by tatarsky** [comment]
Here are the steps:
1. We must get SDSC to bring a 1G fiber drop between your old cluster rack and new rack CC36. This will require these sub-components:
   -A fiber cable which SDSC will have to provide I believe as the length of it is unclear. 
   -Two SR 1G SFP LC connector optics which we will have to purchase
   -We will connect these options between I believe the Cisco Small Business switch in your old cluster Rack and the new 1G switch in rack CC36
   -Given the network ranges are THE SAME we will want to closely check that my efforts to have Advanced HPC not conflict in IP addresses worked as planned.
2. My suggestion is then for any number of node systems you wish to migrate the quickest path is to manually using a CentOS 6 DVD load what is called "minimal" CentOS 6 and assign the same IP address as before. Once the system can be reached as root, I will run the puppet steps (Which I am working on) to convert it from that minimal state to a Lustre Client and SGE Node.

If the systems do not have a DVD drive a USB drive should be usable or we could setup network installs which just takes longer so I would encourage the DVD path for that few nodes.

Is this clear?

**Comment by nariai** [comment]
Thanks, then I guess Hiroko will need to coordinate with SDSC to get cables and connections, and do software configuration. Let's talk about this over the Skype next week.

**Comment by tatarsky** [comment]
I have asked SDSC for an update on the 10G fiber they promised last week. That is NOT the same thing. If you need suggested part numbers for 1G SR SFP LC connectors advise.

**Comment by tatarsky** [comment]
SDSC claims they will have parts next week. They forgot our order.

**Comment by tatarsky** [comment]
My suggestion is once the 10G request is complete that we start a new ticket entry to see if we can get this 1G link. It might be confusing if done at the same time.

**Comment by tatarsky** [comment]
1G request being tracked as its own Git due to SDSC component. #51

**Comment by nariai** [comment]
Hi Paul, we are now ready to start migrating compute nodes from the old cluster to the new cluster. Can you start the process? If there is something that we need to do on-site, please let us know. If you think it's better to have a face-to-face meeting via Skype, please let me know.

**Comment by tatarsky** [comment]
The initial task is in your hands for this one. 

Basically take a gander at "step 2" up a few. 

You will need to download a CentOS 6.6 DVD which I can provide a link to and then pick a node to work on the steps with me. Basically it will be performing a very basic CentOS install, assigning the same address the unit had before and doing what is called a "minimal" install.

But it will ask you a few items such as partition sizes I can review with you on the first one and then just repeat the process.

Once its on the network I take over and puppet does the rest.

When can you start with the DVD on a starter node?

**Comment by tatarsky** [comment]
Grab the following DVD and let me know when you want to go through the steps. Unless by some chance the nodes have an IPMI KVM or something I will only be able to describe these steps to you.

ftp://ftp.uci.edu/mirrors/centos/6.6/isos/x86_64/CentOS-6.6-x86_64-bin-DVD1.iso

**Comment by tatarsky** [comment]
Note clearly this is based on a belief that @hirokomatsui confirmed the nodes all have DVD drives. You will need to attach a keyboard and monitor to the machines but SDSC has lots of those.

**Comment by tatarsky** [comment]
I see some IP addresses for IPMI but I don't seem to be able to connect to them. Probably this would make sense for a Skype to review the options and quickest way forward.

**Comment by hirokomatsui** [comment]
I will try to work on tomorrow, otherwise on Friday. I think I'm ok to install the CentOS and get the network connection by myself. I'll let you know when tomorrow after fixing my schedule.

**Comment by tatarsky** [comment]
Just to be very clear: you do not have to do any changes to the cabling of the network connection to the node. Just remember the IP address of the one you pick to re-install. 

Be sure BTW that the nodes contain no data you wish to save. CentOS will erase it. 

Happy to help, just let me know the best method to do so given the noise of the datacenter.

**Comment by hirokomatsui** [comment]
I'll start working at SDSC 10am tomorrow, installing CentOS on cn19.

**Comment by tatarsky** [comment]
Sounds good. I have an appointment at 3:00PM (1:00PM your time) but I will do my best to help. I am assuming cn19 will come back up on the current IP:

 10.0.0.19/16

You will probably want to set default gateway via a new cluster head node. I would go with:

10.0.16.10    (thats hn1)

And if you wish to enter DNS data for starters:

Domain=local
nameserver 10.0.16.10

But I can handle all that with the puppet run.

**Comment by hirokomatsui** [comment]
Thanks Paul.
BTW, what is your schedule on the next week? We want to move all the old compute nodes to the new cluster before the holiday if it's possible. Please let us know. I'm working at office until next Wednesday.

**Comment by tatarsky** [comment]
I am working the same and happy to help. I will likely take off Thursday and Friday. I will be working the week after.

**Comment by hirokomatsui** [comment]
I'm installing CentOS on cn1 instead of cn19. cn12-19 don't have DVD drive, which we bought later than others.

**Comment by tatarsky** [comment]
OK. If you need my input on any of the install questions, my line is clear: (608)-271-6817

**Comment by tatarsky** [comment]
Do cn12-cn19 have USB ports?

**Comment by tatarsky** [comment]
If so, starting with 6.5 you can burn the ISO to a big enough USB drive.
https://wiki.centos.org/HowTos/InstallFromUSBkey

**Comment by hirokomatsui** [comment]
We can copy the whole disk image of cn1 to the others using ganglia which is a little modified by Microway:
https://flc.ucsd.edu/mcms/
I'll check it later

**Comment by hirokomatsui** [comment]
cn1 is connected. I'll work on the other stuff there and start copying the disk.

**Comment by tatarsky** [comment]
I am on cn1 as root and checking it.

I'm not sure what disk copy method they are using in that link. 

But if that works, sounds reasonable with the following warning:

Be very careful copying images with configured network interfaces or they will conflict. You will have to change the config of a copied image in a few places. 

So far cn1 looks good however.

**Comment by tatarsky** [comment]
Please reboot cn1 when convenient to remove the network manager component. I made the config file change but it requires a reboot and it appears to have dropped. I assume something is being done.

**Comment by hirokomatsui** [comment]
cn1 doesn't have connection for now?

**Comment by tatarsky** [comment]
No. It appears to have dropped. I thought you were working on it. Can it be reset?

**Comment by hirokomatsui** [comment]
rebooted!

**Comment by tatarsky** [comment]
Thanks. I must have fat fingered something. My apologies. Back on the machine and working on Lustre.

Do you want me to hold however as it will be adding a bunch of stuff that probably is unique to the final system....

**Comment by hirokomatsui** [comment]
No, I will leave from the cn1 for now. The old cluster has a system to back up a node, and copy it to the others.
Can you set up the cn1 to migrate to the new cluster, SGE etc if needed. Then, I'll copy it.

**Comment by tatarsky** [comment]
That really isn't going to work. Unique keys and such are created when you deploy many of the items I add. Imaging isn't going to handle that process.

**Comment by tatarsky** [comment]
Basically without knowing anything about this imaging process I would recommend you do the basic installs and let puppet do the heavy lifting.

**Comment by tatarsky** [comment]
Its a pain I know....

**Comment by tatarsky** [comment]
Or if you do want to try one, do it before I add all the items I feel will be needing reinstall if done via clone. Aka now.

**Comment by hirokomatsui** [comment]
OK, I'll start copying. We've been using the system several times when updating OS.

**Comment by tatarsky** [comment]
Give it a try, but I know Puppet, SGE and Lustre make unique items per node that would be best if done in a cloning environment to be done "after clone". Because otherwise I'll just be reinstalling each item. 

If that makes sense.

**Comment by tatarsky** [comment]
OK. Shortly I will have to leave for an appointment. I am waiting on Advanced HPC to install the replacement drive. Do you need anything else from me until the nodes are all converted to CentOS?

**Comment by hirokomatsui** [comment]
After talking with the other, I'll install CentOS to the other nodes instead of copying cn1.
So please go a head to modify cn1.

**Comment by hirokomatsui** [comment]
I will research a bit if I can install CentOS remotely from NFS mounted drive. If I cant, will do onsite next week.

**Comment by tatarsky** [comment]
The CentOS network install process is called Kickstart and I could set up a server for it on one of the head nodes. But I always do it from a CentOS machine. 

But its a bit complex and would require you work with me from a system console to make sure what is called PXE is booted from (basically you boot from a network card) and that it performs the proper steps after talking to the Kickstart server. 

The time this would take is not overly long but may delay getting things done if we have issues compared to the plain old DVD method. 

But given there are 18 more systems I'm willing to give it a go if you can contact me Monday from a node...up to you.

**Comment by hirokomatsui** [comment]
Or I was thinking to modify Grub boot loader.
But probably installing onsite will be easiest way than doing all the research. I'm looking at the way to install CentOS via hard drive, which will be NFS mounted ISO image. In that way, I don't have to push the DVD and USB one by one.

**Comment by tatarsky** [comment]
Basically I'd like to chat when possible in the morning. I'm very familiar with the options and I think we should  at least try a network install process tomorrow. I agree for 20 systems it is probably faster. 

But (and this is why I want to talk) the trick is to make sure we coordinate the process of a PXE boot on one unit first and make sure it is correct. (Before we do the rest). 

That is a bit tricky remotely as that datacenter is loud and you will need to do a network boot of a node which is usually in the BIOS POST process. But if we can attempt one the rest should go fairly fast.

I can start the process of setting up a kickstart server in the morning and when you get in perhaps we can do cn2 as a test. Sound reasonable?

**Comment by tatarsky** [comment]
One question for the morning:

There is a DHCP server on flc with some partial config for two hosts (cn1/cn2). I would like to shut that down (and use DHCP on the new kickstart server) but if you are aware of other systems that need DHCP on your backend network we should be careful.

I'll have the rest of the config ready by morning.

**Comment by tatarsky** [comment]
I have incorporated your choices from the load on cn1 for the kickstart file. I've only changed the `/home` partition to be a `/scratch` partition. As we'll Lustre mount the homedirs.

**Comment by tatarsky** [comment]
Oh, and BTW once the dust clears a bit I am more than happy to explain the entire method involved. Just trying to get these nodes up before the holidays.

**Comment by tatarsky** [comment]
I am working on the DHCP component this morning and from my reading of the motherboard manual the nodes have I believe I can walk you through a fairly simple PXE (network) boot. There may be a simple hot key to do a one time boot from the network or we may have to enter the BIOS to set the network card as the first boot. 

Regardless I am available most of the day although I may have to take a small break to attend an office party. 

But I'd really like to try to get cn2 attempted via the method so the others could be done based on its results.

**Comment by hirokomatsui** [comment]
It's ok to stop the DHCP server on flc. Please let me know the direction whatever I need to do.

**Comment by tatarsky** [comment]
OK. My suggestion is we chat for a few minutes because the process will involve a trip to the datacenter and doing a PXE boot on a node. Do you have time for that now?

Quickest to just call my desk: (608)-271-6817  or I can call you...

**Comment by tatarsky** [comment]
Dhcp is running now on fl-hn1 which is the kickstart server. When ready to attempt cn2 from console should be close to correct...

**Comment by hirokomatsui** [comment]
running to SDSC...

**Comment by tatarsky** [comment]
Sounds good! I am on station with hopefully windows open to speed the attempt.

**Comment by tatarsky** [comment]
I saw it try...

**Comment by hirokomatsui** [comment]
Setting looks good. We need some other file?
![img_1131](https://cloud.githubusercontent.com/assets/15457142/11940089/1c9b185c-a7db-11e5-91b3-048fe1c5c627.JPG)

**Comment by tatarsky** [comment]
Try again. There may have been tcpwrappers in the way.

**Comment by tatarsky** [comment]
Got further that time...it asked for /tftpboot/pxelinux.cfg/0A000002 which is the next boot file...checking

**Comment by tatarsky** [comment]
If it has a menu on the screen, choose the second one...

**Comment by hirokomatsui** [comment]
got it! but isn't it CentOS 6.6?
![img_1132](https://cloud.githubusercontent.com/assets/15457142/11940195/d513b04c-a7db-11e5-9aa7-723116dc10d6.JPG)

**Comment by tatarsky** [comment]
Oh, yeah, thats just a typo from my menu

**Comment by hirokomatsui** [comment]
ok, I'll run the CentOS then.

**Comment by tatarsky** [comment]
I stand corrected. I am showing I am attempting 6.7.

**Comment by tatarsky** [comment]
Lets proceed to see if the install works.

**Comment by tatarsky** [comment]
I may downgrade it once we get the mechanics done.

**Comment by tatarsky** [comment]
I see the install in process

**Comment by tatarsky** [comment]
If it asks a question it means I don't have the kickstart file right

**Comment by tatarsky** [comment]
I am attached to the console. (VNC is activated once we get this far).

**Comment by hirokomatsui** [comment]
not moving from this message for some minuts
![img_1133](https://cloud.githubusercontent.com/assets/15457142/11940356/0228ac30-a7dd-11e5-82cb-b654dd56b6d0.JPG)

**Comment by tatarsky** [comment]
Yep. I am on the console. It is loading the software.

**Comment by tatarsky** [comment]
You cannot see it but I can.

**Comment by hirokomatsui** [comment]
Cool!

**Comment by tatarsky** [comment]
I am working on downgrade to 6.6 for the next test.

**Comment by tatarsky** [comment]
Yep!

**Comment by tatarsky** [comment]
Its almost done with the software. I want to make sure it reboots and comes back up but then I think it would be wise to use 6.6. I forgot that part.

**Comment by hirokomatsui** [comment]
I think so, because all the other nodes are 6.6

**Comment by tatarsky** [comment]
yeah, I forgot that actually.

**Comment by tatarsky** [comment]
Should be fairly quick.

**Comment by tatarsky** [comment]
I believe the system is about to reboot.

**Comment by hirokomatsui** [comment]
Yap, I see that.

**Comment by tatarsky** [comment]
Once I can get to it give me about 20 mins to change from 6.7 to 6.6 and we'll try again.

**Comment by tatarsky** [comment]
Oh, and stop it from booting PXE again.

**Comment by tatarsky** [comment]
Or choose "local disk"

**Comment by hirokomatsui** [comment]
6.7 is start running.

**Comment by tatarsky** [comment]
ok. am going to check it came out ok

**Comment by tatarsky** [comment]
On the machine...so I am now downgrading and we can do it again. But that went pretty well I feel!

**Comment by hirokomatsui** [comment]
I agree, saw 6.7 started and root password has been set correctly.

**Comment by tatarsky** [comment]
Generating repo for 6.6...

**Comment by tatarsky** [comment]
OK, try a PXE again on cn2

**Comment by hirokomatsui** [comment]
OK

**Comment by tatarsky** [comment]
I did that pretty quick but I hope I did it correctly.

**Comment by tatarsky** [comment]
Menu should say 6.6

**Comment by hirokomatsui** [comment]
installing

**Comment by tatarsky** [comment]
I am connected to VNC. It is mkfs.ing scratch

**Comment by tatarsky** [comment]
I believe we will know in a moment and then we can go down the line. I can actually control if the unit boots locally without the menu and leaving them in PXE....

**Comment by tatarsky** [comment]
I have made the other systems config entry to be PXE ready.

**Comment by hirokomatsui** [comment]
I have to change the boot order on the bios setting menu once installing's done.

**Comment by tatarsky** [comment]
You can do that or I can actually tell PXE "don't use a menu, just boot off the local drive"

**Comment by hirokomatsui** [comment]
Probably it's better that I change, then the nodes won't have to go PXE every time booting.

**Comment by tatarsky** [comment]
Fair enough!

**Comment by tatarsky** [comment]
Loading software.

**Comment by tatarsky** [comment]
rebooting.

**Comment by hirokomatsui** [comment]
You've changed the PXE setting as booting from HDD.

**Comment by tatarsky** [comment]
Yep. That still OK?

**Comment by hirokomatsui** [comment]
Sure

**Comment by tatarsky** [comment]
You can change the BIOS to also not boot PXE.

**Comment by tatarsky** [comment]
OK. I show a 6.6 system!

**Comment by tatarsky** [comment]
I think I should have the "/" filesystem a little bigger.

**Comment by tatarsky** [comment]
As it includes /var. I propose 200GB. Its currently this:

```
Filesystem            Size  Used Avail Use% Mounted on
/dev/mapper/vg0-lv_root
                       50G  793M   46G   2% /
tmpfs                  32G     0   32G   0% /dev/shm
/dev/sda1             477M   33M  419M   8% /boot
/dev/mapper/vg0-lv_scratch
                      837G   73M  794G   1% /scratch
```

**Comment by hirokomatsui** [comment]
That's right. Some software need temporary files on there.

**Comment by hirokomatsui** [comment]
10GB for / ?

**Comment by tatarsky** [comment]
Yeah. We don't have a ton of room, but lets take it up a bit.

**Comment by tatarsky** [comment]
OK. Made that change.

**Comment by tatarsky** [comment]
all new builds will get 200GB "/"

**Comment by tatarsky** [comment]
I would say start this one again and/or as many as you want. I feel it looks good.

**Comment by hirokomatsui** [comment]
OK, re-starting on cn2, cn1 then the others.

**Comment by tatarsky** [comment]
Nice!

**Comment by hirokomatsui** [comment]
Will I see the console?

**Comment by tatarsky** [comment]
The menu will appear, but the rest of the install goes to VNC. I can disable that if desired after cn1/cn2 to make sure they come up.

**Comment by hirokomatsui** [comment]
OK, doesn't have to be disabled.

**Comment by tatarsky** [comment]
Up to you....once we prove they rebuild clean I don't need to see the output. I also get a summary of where it is in the syslog.

**Comment by tatarsky** [comment]
I use the VNC when I am debugging a new node platform. As the install will stop and ask questions if I've not provided the proper KS values.

**Comment by tatarsky** [comment]
Formating partitions

**Comment by hirokomatsui** [comment]
I see, am starting cn1 too.

**Comment by tatarsky** [comment]
Cool!

**Comment by tatarsky** [comment]
In theory I suspect you can do about 5-10 at a time. Probably more.

**Comment by tatarsky** [comment]
Then later tonight I will do the Lustre, SGE and puppet. 

I have one errand at 4:00PM my time (2:00PM your time)

**Comment by hirokomatsui** [comment]
OK, sounds great!

**Comment by tatarsky** [comment]
Hopefully this is a little easier ;)

**Comment by hirokomatsui** [comment]
Much easier for me.

**Comment by tatarsky** [comment]
And I will detail the config of the Kickstart items for folks once we get this done. It comes in very handy.

**Comment by tatarsky** [comment]
cn2 is loading software.

**Comment by tatarsky** [comment]
cn1 still doing disk partition formats but almost done.

**Comment by tatarsky** [comment]
cn2 rebooting.

**Comment by hirokomatsui** [comment]
working on cn3

**Comment by tatarsky** [comment]
yep.

**Comment by tatarsky** [comment]
I am switching the PXE default to the local disk so you don't have to play "find the console" as quickly ;)

**Comment by tatarsky** [comment]
New partitions give us a bit more breathing room.

```
Filesystem            Size  Used Avail Use% Mounted on
/dev/mapper/vg0-lv_root
                      197G  801M  186G   1% /
tmpfs                  32G     0   32G   0% /dev/shm
/dev/sda1             477M   33M  419M   8% /boot
/dev/mapper/vg0-lv_scratch
                      689G   69M  654G   1% /scratch
```

**Comment by tatarsky** [comment]
I would say "lets proceed"

**Comment by tatarsky** [comment]
cn3 is formatting filesystems
cn1 is rebooting

**Comment by tatarsky** [comment]
I am actually going to do my errand but I will have my laptop as it involves another project. 
I show cn1 looking good as well. 
cn3 loading software

**Comment by tatarsky** [comment]
I have all the other nodes configured to accept kickstart. I will attempt to keep up with the menu change but you may find some of them waiting if you don't flip them back to BIOS boot off the local drive.

**Comment by hirokomatsui** [comment]
OK. cn5's on VNC. cn4 might have HDD issue, am checking.

**Comment by hirokomatsui** [comment]
cn4's on VNC

**Comment by hirokomatsui** [comment]
cn6's on VNC

**Comment by hirokomatsui** [comment]
cn8's on VNC.
cn7 cannot boot up. We will have to discard it by hardware issue.
I'm leaving for a lunch break, and will do the rest after coming back.

**Comment by tatarsky** [comment]
Sound good. Removed boot triggers for all but cn7

**Comment by hirokomatsui** [comment]
cn9's on VNC

**Comment by hirokomatsui** [comment]
cn10's on VNC

**Comment by hirokomatsui** [comment]
cn11's on VNC

**Comment by hirokomatsui** [comment]
cn12's on VNC

**Comment by tatarsky** [comment]
Noted!

**Comment by tatarsky** [comment]
Moved menu item aside for less than cn12

**Comment by hirokomatsui** [comment]
cn13's on VNC

**Comment by hirokomatsui** [comment]
cn14's on VNC

**Comment by hirokomatsui** [comment]
cn15's on VNC

**Comment by hirokomatsui** [comment]
cn16's on VNC

**Comment by tatarsky** [comment]
cn16 is loading software. Removed menus from < cn16

**Comment by tatarsky** [comment]
cn17 is formatting filesystems

**Comment by tatarsky** [comment]
cn18 loading software

**Comment by hirokomatsui** [comment]
cn19's on VNC. I'm leaving soon

**Comment by tatarsky** [comment]
c19 formatting drives.

**Comment by tatarsky** [comment]
I may have messed up the link for cn20....is there a cn20?

**Comment by hirokomatsui** [comment]
no cn19. we have only 19

**Comment by hirokomatsui** [comment]
I meant no cn20

**Comment by tatarsky** [comment]
Gotcha!

**Comment by tatarsky** [comment]
So then I've only left cn7 which I know is down.

**Comment by tatarsky** [comment]
rest look great! I will be back shortly and will work on the details. Thank you very much for the effort!

**Comment by tatarsky** [comment]
So only issues this morning I see as I work on their final config is `cn7` is down (known) and `cn12` which was up appears down. It may be sitting at a PXE menu and just need a reset. 

Are these units labelled externally for use with tickets with SDSC?

I am working on deployment of them. Estimate some tests probably this morning could be run.

**Comment by hirokomatsui** [comment]
Yes, all the nodes are labeled and SDSC can restart them.

**Comment by tatarsky** [comment]
We could try asking for them to reset just `c12`. All the other nodes are up and running. 

Remind me the rack number the old nodes are in?

**Comment by tatarsky** [comment]
Ah, I believe the old cluster is in `K36`

**Comment by hirokomatsui** [comment]
Yes, it's K36

**Comment by tatarsky** [comment]
Sent a ticket!

**Comment by tatarsky** [comment]
`cn12` is sick in some way I cannot determine. Perhaps next time down there can hook up a console.
I have to do another round of errands shortly so maybe in the New Year.

**Comment by hirokomatsui** [comment]
OK. I'll look at it later.

**Comment by tatarsky** [comment]
Oh, and I have not fully automated this process to all nodes but from the headnodes the pdsh utility should work with a group of "cn"

```
 pdsh -g cn uptime
cn1:  13:11:35 up 18:55,  0 users,  load average: 0.00, 0.04, 3.09
cn2:  13:11:35 up  7:58,  0 users,  load average: 0.00, 0.00, 0.05
cn6:  13:11:35 up  8:05,  0 users,  load average: 0.16, 0.08, 0.09
cn11:  13:11:35 up  7:28,  0 users,  load average: 0.00, 0.07, 2.53
cn3:  13:11:35 up  9:01,  0 users,  load average: 0.00, 0.00, 0.07
cn16:  13:11:35 up  6:32,  1 user,  load average: 0.07, 0.08, 2.73
cn14:  13:11:35 up  6:16,  0 users,  load average: 0.11, 0.17, 4.19
cn10:  13:11:35 up  7:44,  0 users,  load average: 0.08, 0.10, 0.76
cn17:  13:11:35 up  6:04,  0 users,  load average: 0.09, 0.34, 5.11
cn4:  13:11:35 up  6:31,  0 users,  load average: 0.11, 0.04, 0.01
cn15:  13:11:35 up  6:32,  0 users,  load average: 16.30, 16.16, 16.77
cn5:  13:11:35 up  8:07,  0 users,  load average: 0.21, 0.12, 0.09
cn9:  13:11:35 up  7:56,  0 users,  load average: 0.07, 0.03, 0.04
cn8:  13:11:35 up  6:26,  0 users,  load average: 0.13, 0.07, 2.40
cn19:  13:11:35 up  5:56,  1 user,  load average: 15.14, 18.04, 23.77
cn13:  13:11:35 up  7:32,  0 users,  load average: 0.09, 0.09, 3.07
cn18:  13:11:35 up 22:14,  0 users,  load average: 0.12, 0.63, 6.82
```

I have removed for now cn7 and cn12 from that group. The newer nodes is group `n` aka `pdsh -g n`
Advise if you get asked for a key. I need to do some automation of that deployment and it may be missing in some systems.

**Comment by tatarsky** [comment]
I am going to track potential next steps for this under #83.
The nodes themselves are converted but some changes to their hardware would make them much more useful.
